CHANGELOG.rst

```diff
@@ -2,6 +2,191 @@
 Release Notes
 =============
 
+.. _Release Notes_7.42.0:
+
+7.42.0 / 6.42.0
+======
+
+.. _Release Notes_7.42.0_Prelude:
+
+Prelude
+-------
+
+Release on: 2023-01-19
+
+- Please refer to the `7.42.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7420>`_ for the list of changes on the Core Checks
+
+
+.. _Release Notes_7.42.0_Upgrade Notes:
+
+Upgrade Notes
+-------------
+
+- Downloading and installing official checks with `agent integration install`
+  is no longer supported for Agent installations that do not include an embedded
+  python3.
+
+- Starting with Agent 7.40, the Agent will fail to start when it is unable to determine the hostname, instead of silently using an irrelevant hostname (usually, a container ID).
+  Hostname resolution is key to many features and failure to determine hostname means that the Agent is not configured properly.
+  This change mostly affects Agents running in containerized environments as we cannot rely on the OS hostname.
```

Comment:
Let's delete this, it's not meant to be added to 7.42's changelog (see https://github.com/DataDog/datadog-agent/pull/14249/)

Reply:
Removed